src/shortcuts.ts

```diff
@@ -0,0 +1,19 @@
+export interface ShortcutsHook {
+  getShortcutDefinitions(): ShortcutDefinition[]
+}
+
+export interface ShortcutDefinition {
```

Comment:
Did you consider adding a `getText` method?  I'm wondering if that would meet the requirements you're aiming for below (rather than making a confident suggestion on a change).

> The positions are extended with a new availableShortcutIds property which allows defining which shortcuts can be applied to them. Underlying tokens get a new optional category property which allows giving more meaning to what they are.
Specifically here, the claimable category tells the wallet the tokens and amounts which can be claimed.

Reply:
Thanks for the suggestion, one drawback of adding a `getText` method is it puts more constraints for the display done by the wallet. The wallet already has good components for displaying amounts correctly with the right local currency and decimal symbol (based on the user preferences). Also what if we wanted to make one part of the string bold or something?
We'd need to pass more user context here to be able to do that. Or find a way to represent that with a custom syntax.

For now I preferred letting the wallet use the full position object and decide how it wants to display it.
I had to add the `category` type though to the tokens so the wallet can tell which part is the reward. 
But I see other benefits of this for future features. We could for instance use this info to create a breakdown of value: total claimable, total debt, total collateral, etc
Or if we build a detail screen for positions, we could add a visual difference to the claimable tokens (+ an action button there directly for the shortcut).